public/app/features/dashboard/dashgrid/DashboardEmpty.tsx

```diff
@@ -148,22 +174,30 @@ function getStyles(theme: GrafanaTheme2) {
       padding: theme.spacing.gridSize * 4,
     }),
     others: css({
+      width: '100%',
       label: 'others-wrapper',
       alignItems: 'stretch',
       flexDirection: 'row',
       gap: theme.spacing.gridSize * 4,
 
-      [theme.breakpoints.down('sm')]: {
+      [theme.breakpoints.down('md')]: {
```

Comment:
Why do we change the breakpoint here? Doesn't fit well 3 sections in an iPad?

Reply:
exactly, it ends up being too small and breaking the responsive behavior 
![image](https://github.com/grafana/grafana/assets/11707172/8aaa9f03-359b-4865-8780-93f89345262a)